README.md

```diff
@@ -105,29 +105,29 @@ For example:
 var x: number = 1;
 ```
 
-This is not valid JavaScript code, because it contains a so-called type annotation. When the TypeScript Compiler parses this code to produce a TypeScript AST, that `: number` syntax will be represented in the tree, and this is simply not something that ESLint can understand without additional help.
+This is not valid JavaScript code, because it contains a so-called type annotation. When the TypeScript Compiler parses this code to produce a TypeScript AST, that: number syntax will be represented in the tree, and this is simply not something that ESLint can understand without additional help.
```

Comment:
Leave this as is - it's referring to the type annotation syntax.

```suggestion
This is not valid JavaScript code, because it contains a so-called type annotation. When the TypeScript Compiler parses this code to produce a TypeScript AST, the `: number` syntax will be represented in the tree, and this is simply not something that ESLint can understand without additional help.
```